frontend/src/components/CardTitle/CardTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Typography,} from "@material-ui/core";
+import {APP_NAME} from "../../constants/constants";
+import {VerticalSpace} from "../Shared/VerticalSpace"
+import {Centered} from "../Shared/Centered";
+import styled from "styled-components";
+
+export type TitleProps = {
+    readonly title: string | undefined;
+};
+
+const CardTitleComponent = (props: TitleProps | undefined) => {
+    const titleComponent = (
+        <>
+            <Centered>
+                <Typography component="h5" variant="h5">
+                    {props?.title || APP_NAME}
+                </Typography>
+                <VerticalSpace height="2rem"/>
```

Comment:
Nie wiem czy to jest dobra praktyka :/ Czy nie lepiej uzyc czegos responsywnego. To ja tak o sobie zrobiłem. Ale mam nadzieje, że Wy, a szczególnie mistrz CSSa wiece co robicie :)